src/wizard/unity/index.md

```diff
@@ -9,6 +9,8 @@ type: framework
 
 Get the SDK via the [Unity Package Manager using a Git URL](https://docs.unity3d.com/Manual/upm-ui-giturl.html) to Sentry's SDK repository:
 
+> Make sure not empty space is appended to the end of the URL, otherwise Unity fails to find the package.
```

Comment:
I'm up for suggestions on how to phrase this and what format. I just to make sure no one will have the same bad experience I did just because of a bad copy and paste

Reply:
submit a PR to UPM, url.trim() :D